scss/modules/dropzone.scss

```diff
@@ -3,285 +3,339 @@
-      margin-top: -$dropzoneHeight - 20px;
-    }
-  }
+    .upload-items {
+        height: $dropzoneHeight;
+        padding: 20px 0 0 20px;
+        position: relative;
+        z-index: 5;
+        @include clear-float;
 
-  &.small {
-    padding-bottom: 0;
-    height: $smallHeight;
-    @include single-transition(height, $transitionDuration);
+        .item {
+            width: $previewItemWidth;
+            height: 90px;
+            border: 1px solid $white;
+            background: $white;
+            border-radius: $borderRadiusSmall;
+            margin-right: 20px;
+            margin-bottom: 20px;
+            float: left;
+
+            .success-content {
+                display: none;
+            }
+            .loading-content {
+                padding-top: 20px;
+            }
+            .file-size {
+                text-align: center;
+                font-size: $smallestFontSize;
+                line-height: 10px;
+                color: $grayLight;
+                padding-top: 15px;
+                padding-bottom: 5px;
+                * {
+                    font-weight: normal;
+                }
+            }
+            .progress {
+                width: 50px;
+                height: 6px;
+                margin: 0 auto;
+                background: $grayLighter;
+                border-radius: 3px;
+                overflow: hidden;
+                .bar {
+                    height: 100%;
+                    background: $blue;
+                    width: 0%;
+                }
+            }
+            .icon {
+                height: 25px;
+                color: $grayLight;
+                font-size: 25px;
+                text-align: center;
+                display: block;
+                animation: ROTATE-ANIMATION 1s linear infinite;
+                -webkit-animation: ROTATE-ANIMATION 1s linear infinite;
+                -moz-animation: ROTATE-ANIMATION 1s linear infinite;
+                -o-animation: ROTATE-ANIMATION 1s linear infinite;
+            }
+            .image {
+                padding: 10px 10px 5px 10px;
+                height: 65px;
+                line-height: 45px;
+                text-align: center;
+                img {
+                    vertical-align: bottom;
+                    max-width: 100%;
+                    max-height: 100%;
+                }
+            }
+            .tick {
+                text-align: center;
+                color: $grayLight;
+                font-size: 15px;
+                line-height: 15px;
+                display: block;
+            }
 
-    .addition {
-      display: none;
-    }
-    .description {
-      height: $smallHeight;
-
-      .icon {
-        font-size: 25px;
-        line-height: 25px;
-      }
-      .title {
-        font-size: $dropzoneSmallFontSize;
-        line-height: $dropzoneSmallFontSize;
-      }
-    }
-    .upload-items {
-      height: $smallHeight;
+            &.dz-success {
```

Comment:
That's a class provided by the dropzone plugin (http://www.dropzonejs.com/). We can't really influence that.